core/src/CabanaCore_config.hpp.cmakein

```diff
@@ -1,8 +1,10 @@
 #ifndef CABANA_CORE_CONFIG_HPP
 #define CABANA_CORE_CONFIG_HPP
 
-#define Cabana_VERSION_STRING "@PROJECT_VERSION@"
+#define CABANA_VERSION_STRING "@PROJECT_VERSION@"
 
-#define Cabana_GIT_COMMIT_HASH "@Cabana_GIT_COMMIT_HASH@"
+#define CABANA_GIT_COMMIT_HASH "@Cabana_GIT_COMMIT_HASH@"
+
+#define CABANA_ENABLE_MPI @Cabana_USE_MPI@
```

Comment:
```suggestion
# cmakedefine01 CABANA_ENABLE_MPI
```

Reply:
good call - cleaned this up and committed with you as co-author